labs/firepower-restapi-105/2.md

```diff
@@ -1,27 +1,24 @@
 ## Problem: REST operation fails with HTTP 403
-This may due to many access and privileges related reasons.
+This may due to many access and privileges-related reasons.
 
-Login into the FMC UI
-For your own setup:
-https://hostnameorIPaddressofyourFMC
+Login into the FMC UI via [your own setup](https://hostnameorIPaddressofyourFMC) if you are using the [DevNet Sandbox](https://fmcrestapisandbox.cisco.com).
 
-incase if you are using devnet sandbox https://fmcrestapisandbox.cisco.com
+Use the same user credentials, whoever's authentication token you are using for API. Attempt the same operation using UI. If it fails, then you need to request your admin to change access privileges of the user.
 
-Use the same user credentials; whose authentication token you are using for API. Attempt the same operation using UI. If it fails, then you need to request your admin to change access privileges of the user.
+Make sure the URL in the request has the right **domain** in which the *user* is authorized.
 
-Make sure the url in the request has the right **domain** in which the *user* is authorized.
+**Note:** DevNet Sandbox may have restrictions on privileges.
 
-**Note:** DevNet sandbox may have restrictions on privileges.
 
 ## Problem: A REST operation fails with http 401
-Most likely the authentication token you are using for the API is expired. You have to refresh the token. Please read more on refresh token in the Learning lab FMC REST API 102.
+Most likely the authentication token you are using for the API has expired. You have to refresh the token. Please read more on refresh token in the [Learning Lab FMC REST API 102](https://learninglabs.cisco.com/lab/ucsd-rest-api-102/step/1).
```

Comment:
Is a "refresh token" a type of token, so that "refresh" is the modifier? Or should it be rephrased as, "Please read more about refreshing tokens in the..." Plus I don't read anything about tokens in the linked lab... hm. Oh, that link goes to a UCSD lab, do you mean to link to an FMC REST API lab?